src/app/modules/entry-live/views/live-discovery-table/users-table/live-discovery-users-table.config.ts

```diff
@@ -21,7 +21,7 @@ export class LiveDiscoveryUsersTableConfig extends ReportDataBaseConfig {
         };
       })
       .sort((a, b) => b.count - a.count);
-    return topFlavors.length ? topFlavors[0].name : 'N/A';
+    return topFlavors.length ? (topFlavors[0].count > 0 ? topFlavors[0].name : 'N/A') : 'N/A';
```

Comment:
@Adirv Good catch. I'd rather have it without nested conditional operators:
`return topFlavors.length && topFlavors[0].count ? topFlavors[0].name : 'N/A';`

Reply:
Thanks @amirch1 , Nadav pointed me to the right direction :)
I will make the changes now.